### PR TITLE
 Hyper-V: Fix 2 issues of Get-NetIPAddress and CreateIfupConfigFile

### PR DESCRIPTION
**Pull request by huijuanzhao**
Fix the issue that run Get-NetIPAddress failed when run it in remote server. 
Before fix, only can run it in local server, but if run it in remote server, will get wrong IP/interface as it still get the local IP/interface.
After fix, "Invoke-Command -ComputerName $HvServer" can run successful both in local and remote server as there is parameter $HvServer which can specify the right server.

### CODE REVIEW

**Comment by huijuanzhao** [comment]
Before fix the script:
Errors in output:
09/27/2019 01:43:10 : [ERROR] EXCEPTION: You cannot call a method on a null-valued expression.
09/27/2019 01:43:10 : [ERROR] Source: Line 1576 in script .\Libraries\CommonFunctions.psm1.

[LISAv2 Test Results Summary]
Test Run On           : 09/27/2019 01:42:11
VHD Under Test        : D:\RHEL-8.1.0-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el8.x86_64
Final Kernel Version  : xxxx.el8.x86_64
Total Test Cases      : 1 (0 Passed, 1 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:4

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              NET-INTERNAL                                                                      FAIL                 2.92 



After fix the script:
[LISAv2 Test Results Summary]
Test Run On           : 09/27/2019 01:57:08
VHD Under Test        : D:\RHEL-8.1.0-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el8.x86_64
Final Kernel Version  : xxxx.el8.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:4

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              NET-INTERNAL                                                                      PASS                 2.48

**Comment by huijuanzhao** [comment]
For another fix file utils.sh:
Fix the CreateIfupConfigFile to be available both for RHEL_7 and RHEL_8.  Delete "NM_CONTROLLED=no" as there is only NetworkManager to handle the network as default in RHEL_8.

Before fix the script:
Test failed in RHEL_8.
Errors in output:
09/27/2019 02:54:06 : [ERROR] Could not ping from 10.10.10.1 to 10.10.10.2

After fix the script:
1. RHEL_8 test results:
[LISAv2 Test Results Summary]
Test Run On           : 09/27/2019 08:22:10
VHD Under Test        : D:\RHEL-8.1.0-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el8.x86_64
Final Kernel Version  : xxxx.el8.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:6

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              NET-PRIVATE                                                                       PASS                 3.62

2. RHEL_7 test results:
[LISAv2 Test Results Summary]
Test Run On           : 09/27/2019 08:29:16
VHD Under Test        : D:\RHEL-7.8-xxxx-x86_64-GEN2-A.vhdx
Initial Kernel Version: xxxx.el7.x86_64
Final Kernel Version  : xxxx.el7.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:5

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              NET-PRIVATE                                                                       PASS                 3.83

File: Testscripts/Linux/utils.sh [hunk 1]
```diff
@@ -1368,7 +1368,6 @@ CreateIfupConfigFile()
 						BOOTPROTO=none
 						IPADDR="$__ip"
 						NETMASK="$__netmask"
-						NM_CONTROLLED=no
```
Comment (LiliDeng) [inline Testscripts/Linux/utils.sh:1371]: what is the usage of this setting? does this change will affect other version of RHEL? like RHEL6.X?

Comment (huijuanzhao) [reply Testscripts/Linux/utils.sh:1371]: No, will not affect other version of RHEL.

Comment (huijuanzhao) [reply Testscripts/Linux/utils.sh:1371]: The first fix has some issue, will pull request later after fix, sorry for the noise.

**Pull request closed without merging**